tests/nlu/tokenizers/test_whitespace_tokenizer.py

```diff
@@ -152,3 +163,47 @@ def test_whitespace_training(supervised_embeddings_config):
     assert examples[1].data.get(TOKENS_NAMES[TEXT])[0].text == "i"
     assert examples[1].data.get(TOKENS_NAMES[TEXT])[1].text == "want"
     assert examples[1].data.get(TOKENS_NAMES[TEXT])[2].text == "tacos"
+    assert examples[2].data.get(TOKENS_NAMES[ACTION_NAME])[0].text == "action"
+    assert examples[2].data.get(TOKENS_NAMES[ACTION_NAME])[1].text == "restart"
+    assert examples[2].data.get(TOKENS_NAMES[TEXT])[0].text == "action_restart"
+    assert examples[2].data.get(TOKENS_NAMES[ACTION_TEXT]) is None
+    assert examples[3].data.get(TOKENS_NAMES[ACTION_TEXT])[0].text == "where"
+    assert examples[3].data.get(TOKENS_NAMES[ACTION_TEXT])[1].text == "are"
+    assert examples[3].data.get(TOKENS_NAMES[ACTION_TEXT])[2].text == "you"
+    assert examples[3].data.get(TOKENS_NAMES[ACTION_TEXT])[3].text == "going"
+
+
+def test_whitespace_processing_with_attribute(supervised_embeddings_config):
+    message = Message(
+        "Any Mexican restaurant will do",
+        {
+            "intent": "restaurant_search",
+            "entities": [
+                {"start": 4, "end": 11, "value": "Mexican", "entity": "cuisine"}
+            ],
+        },
+    )
+    expected_tokens_intent = ["restaurant_search"]
+    expected_tokens_text = ["any", "mexican", "restaurant", "will", "do"]
+    component_config = {"case_sensitive": False}
+    tk = WhitespaceTokenizer(component_config)
+    tk.process(message, INTENT)
+    tokens_intent = message.get(TOKENS_NAMES[INTENT])
+    tk.process(message, TEXT)
+    tokens_text = message.get(TOKENS_NAMES[TEXT])
+    assert [t.text for t in tokens_intent] == expected_tokens_intent
+    assert [t.text for t in tokens_text] == expected_tokens_text
+
+    message = Message(
+        "Where are you going?",
+        {ACTION_NAME: "Where are you going?", ACTION_TEXT: "Where are you going?",},
```

Comment:
```suggestion
        {ACTION_NAME: None, ACTION_TEXT: "Where are you going?"},
```

Reply:
I would like to leave the examples as is -- in reality we will not have the text loaded into action name but I wanted to test that both attributes will be processed correctly.